Realm/RLMHandover.h

```diff
@@ -0,0 +1,67 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2016 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+#import <Foundation/Foundation.h>
+
+@class RLMRealm;
+
+NS_ASSUME_NONNULL_BEGIN
+
+/// A Realm-bound object that can only be passed between threads by exporting for handover
+@protocol RLMThreadConfined <NSObject>
+
+/// The destination `RLMRealm` the object was imported into
```

Comment:
This comment is actually entirely incorrect. I modified the wrong one in response to @bdash's comment. Thanks for catching that.